### PR TITLE
Add travis

### LINKED ISSUE #4: Plug in Travis / Tox and test utilities

### PR DESCRIPTION
**Pull request by ametaireau**
Fix #4

### CODE REVIEW

**Comment by leplatrem** [comment]
r+

*In case you had this in mind, don't forget the badges, maybe coveralls etc.*

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[tox]
+envlist = py27,flake8
```
Comment (Natim) [inline tox.ini]: Can you add support for Python3, this is really important to detect ``UnicodeDecodeError`` with Python2.7 upfront.

**Comment by Natim** [comment]
We can add Py3 support letter it doesn't block this PR.

**Pull request merged**